tests/e2e/Services/Teams/TeamsBaseServer.php

```diff
@@ -281,4 +282,101 @@ public function testDeleteUserUpdatesTeamMembershipCount($data)
         $this->assertIsInt($response['body']['total']);
         $this->assertEquals(true, $dateValidator->isValid($response['body']['$createdAt']));
     }
+
+    public function testTeamDeleteUpdatesUserMembership()
+    {
+        $new_users = [];
+
+        /**
+         * Create 5 new users and add their IDs to an array
+         */
+        for ($i = 0; $i < 5; $i++) {
+            $user = $this->client->call(Client::METHOD_POST, '/users', array_merge([
```

Comment:
Can we just create the team membership without creating the user first?

Reply:
Yeah using the create membership endpoint to create users is fine as well. I just thought it would be better to create the user separately. If it's somewhat of a deal-breaker in terms of test performance, I'll go ahead and do that instead. Let me know!

Reply:
Ya, our tests take a long time as it is so minimizing as much as possible would be good.